server/src/core/server/services/users/delete.ts

```diff
@@ -131,7 +133,8 @@ async function moderateComments(
   filter: FilterQuery<Comment>,
   targetStatus: GQLCOMMENT_STATUS,
   now: Date,
-  isArchived = false
+  isArchived = false,
+  rejectionReason?: Moderate["rejectionReason"]
```

Comment:
Should the type here be `GQLRejectionReason`?

Reply:
✅